spec/test_utils.rb

```diff
@@ -25,18 +26,7 @@ def TestUtils.tokenize(input_string)
   def TestUtils.tokenize_as_list(input_string)
     token_iterator = tokenize(input_string)
 
-    token_list = []
-
-    while true
-      token = token_iterator.next
-      token_list.push token
-
-      if token.token_type == EOF
-        break
-      end
-    end
-
-    token_list
+    token_iterator.map { |token| token }
```

Comment:
okay, `map` is nice.  what about going one step further and adding `to_list` which just does this?